_layouts/datagovsg-v2-search.html

```diff
@@ -0,0 +1,229 @@
+---
+layout: skeleton
```

Comment:
only the egaz repo will use this specific layout?

Reply:
Yep, and also any future repos which are using the new dgs database format